web/src/views/SubmissionPortal/HarmonizerView.vue

```diff
@@ -225,6 +228,12 @@ export default defineComponent({
         ...tabsValidated.value,
         [activeTemplateKey.value]: valid,
       };
+
+      let allTabsValid = true;
+      Object.values(tabsValidated.value).forEach((value) => {
+        allTabsValid = allTabsValid && value;
+      });
+      snackbar.value = allTabsValid;
```

Comment:
This would be a good place to use [Array.prototype.every](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every)
```suggestion
      snackbar.value = Object.values(tabsValidated.value).every((value) => value);
```
Feel free to ignore/modify if you don't think this is more readable.

Reply:
I think that's a reasonable fix. I think it's definitely less cluttered and still readable.

Reply:
This is implemented now